src/image_copy/main.go

```diff
@@ -119,7 +119,16 @@ func mustHaveArchitecture(imageTag, platform, expectedArch string) {
 		log.Fatal("no binaries were found")
 	}
 	for _, file := range files {
-		out = execCmd("file", filepath.Join(tmpDir, file.Name()))
+		fp := filepath.Join(tmpDir, file.Name())
```

Comment:
full path and file pointer both lead to this acronym. Not a big deal, just a thought,=.